web/themes/joinup/joinup_theme.theme

```diff
@@ -1052,6 +1052,22 @@ function joinup_theme_theme_suggestions_container_alter(array &$suggestions, arr
       $suggestions[] = 'container__edit_actions__search_bar';
     }
   }
+
+  // Early bail out.
+  if (!isset($variables['element']['#extra_suggestion'])) {
```

Comment:
Please make like this:
```
/**
 * Implements hook_theme_suggestions_HOOK_alter() for container.html.twig.
 */
function joinup_theme_theme_suggestions_container_alter(array &$suggestions, array $variables) {
  $has_extra_suggestions = isset($variables['element']['#extra_suggestion']);

  if (isset($variables['element']['#id'])) {
    $suggestions[] = 'container__' . str_replace('-', '_', $variables['element']['#id']);

    // Suggestion for search form submit container.
    if (isset($variables['element']['submit']['#id']) && strpos($variables['element']['submit']['#id'], 'edit-submit-search') !== FALSE) {
      $suggestions[] = 'container__edit_actions__search_bar';
    }
  }

  if ($has_extra_suggestions) {
    switch ($variables['element']['#extra_suggestion']) {
      case 'container_grid':
        $suggestions[] = 'container__grid';
        break;

      case 'container_grid_item':
        $suggestions[] = 'container__grid_item';
        break;
    }
  }
}
```

This way:
- you don't stop the function preventing others to include further logic
- you make the check very visible and readable
- you make the check reusable for others later